src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx

```diff
@@ -25,7 +25,9 @@ export function getQuery(view, location) {
   const fields = [];
   const groupby = view.data.groupby ? [...view.data.groupby] : [];
 
-  view.data.fields.forEach(field => {
+  const viewFields = get(view, 'data.fields', []);
```

Comment:
we're not guaranteed that `view.data.fields` is an array.

Reply:
If it is a string should we cast into an array?

Reply:
AFAIK, `view.data.fields` is a nullable array of strings according to this proptype definition:

https://github.com/getsentry/sentry/blob/76cf39a5db99b24322863824d5e2c50311bbb3df/src/sentry/static/sentry/app/sentryTypes.jsx#L113-L122